### PR TITLE
[editor] Updates to  ASR prompt Schema

### PR DESCRIPTION
**Pull request by Ankush-lastmile**
[editor] Updates to  ASR prompt Schema

Adding in changes to the ASR prompt schema. Previously this schema was shipped to unblock.

Couple of changes:

- Support a list of 5 audio formats.
- - Top 5 chosen by [asking GPT](https://lastmileai.dev/workbooks/clr6yqzzy0015pe4ewvgm4ce9)
- - Note: ASR supports FFmpeg to transcribe audio. For now we are choosing a few to start
- Added a few completion params, found while implementing model parser for ASR
- Added a few params for pipeline creation.

TODO:
investigate these types because they may not be primitives
- pipeline creation:
- - "decoder",
- - - I believe this should be an in memory object
- - "feature_extractor"
- - - Same with this, this might have to be an in memory object
- completion params:
- - "generate_kwargs"
- - This is a dictionary/map of ad-hoc parameters

## Testplan

1. run server with model parser registered:
Screenshot 2024-01-09 at 10 50 22 PM

render:

Screenshot 2024-01-09 at 10 49 50 PM

### CODE REVIEW

**Review by rholinshead — APPROVED** [review]

**Review by rholinshead — CHANGES_REQUESTED** [review]
Whoops, sorry. Accepted accidentally

File: plugin.py
Comment (rholinshead) [inline plugin.py:1]: Should remove this

**Review by rholinshead — APPROVED** [review]
Schema looks good, but will probably have merge conflicts with https://github.com/lastmile-ai/aiconfig/pull/845/files so please rebase and make sure those changes are still in.

Otherwise, please remove the plugin.py used for testing

**Comment by Ankush-lastmile** [comment]
rebased and removed `plugin.py` - sorry about that!

**Pull request merged**